README.md

```diff
@@ -66,6 +66,19 @@ Running that will produce something like
     filename: Ace Rimmer to the Rescue-WKsjaOqDXgg.flv
     format id: 34
 
+You can use an array of urls to produce an array of response objects with matching array index (e.g. the 1st response object will match the first url etc...)
+```javascript
```

Comment:
Not sure what the standard is? The README's standard is to use `javascript` up until this point, but is uses `js` after that